config/backend_connections.go

```diff
@@ -61,6 +76,28 @@ func (c *RethinkDBConnection) GetSession() *r.Session {
 	return c.session
 }
 
+// loadCertificate reads a certificate either from a file or from a string
+func loadCertificate() []byte {
+	var cert []byte
+	certPath := viper.GetString("RETHINK_TLS_CERT_PATH")
+	if certPath != "" {
+		var err error
+		cert, err = ioutil.ReadFile(certPath)
+		if err != nil {
+			logrus.Debugf("Could not read rethinkdb certificate: %s", err)
```

Comment:
this should be a warning because a path was given but an error occurred. if you think `Debugf` makes more sense here, we should change that in all locations